torch/csrc/jit/codegen/cuda/scheduler/mma_utils.cpp

```diff
@@ -272,7 +290,7 @@ std::vector<IterDomain*> getMmaRootDimensions(
   // Loop through tensorview's root domains and accumulate all the
   //  root domain IterDomain's that maps to any of the collected
   //  mma root dimension from the mma accumulator tv.
-  for (auto tv_id : tv->getMaybeRFactorDomain()) {
+  for (auto tv_id : tv->getRootDomain()) {
```

Comment:
Also skeptical here of blanket grabbing the root domain. Seems we would want to track through root and rfactor domains for mappings.

Reply:
This would need to be rfactor domain too in case a view is in prolog. Will update. Thanks.